src/app/schedule-view/schedule-event/component.ts

```diff
@@ -10,8 +10,23 @@ import { ScheduleEvent } from '../../models/schedule-event.model';
   providers: [
     // ConstantsService,
   ],
+  host: {
+    '[style.height]': `getHeight()`,
+    '(mouseenter)': `changeHover(true)`,
+    '(mouseleave)': `changeHover(false)`
+  },
 })
 
 export class ScheduleEventComponent{
 	@Input() scheduleEvent: ScheduleEvent;
-}
\ No newline at end of file
+    @Input() normalHeight: number;
+
+    hover: boolean = false;
+
+    changeHover(newHover: boolean) {
+        this.hover = newHover;
+    }
+    getHeight() {
+        return this.hover ? 'auto' : this.normalHeight + 'px';
```

Comment:
Great thanks! Looks good, can we just add (return) type annotations to these functions, and use two-space indentation @sjhuang26 ? Then I will merge this.